content/en/docs/collector/trace-receiver.md

```diff
@@ -66,6 +66,7 @@ with the following command:
 docker run -d --name jaeger \
   -p 16686:16686 \
   -p 14268:14268 \
+  -p 14250:14250 \
```

Comment:
Why is exposing this port necessary?

Reply:
https://github.com/open-telemetry/opentelemetry.io/blob/f9c3c0beb66fbd71c6323d55b88d5394227bc569/content/en/docs/collector/trace-receiver.md?plain=1#L83-L107

Because this port `14250` is needed by `jaeger` at line 97 of this config file.